notebook2/notebook/notebook.py

```diff
@@ -497,26 +420,28 @@ def wait_websocket(ws):
     if notebook is None:
         return
 
-    pod_name = notebook['pod_name']
-    svc_name = notebook['svc_name']
+    pod_uuid = notebook['pod_uuid']
+
+    url = external_url_for('')
+    url = f'{url}instance-ready/{pod_uuid}/'
```

Comment:
`external_url_for(f'instance-ready/{pod_uuid}')` doesn't work?

Reply:
Totally works, I'm trying to keep line length to something like 80, although really should be more consistent.